views/js/qtiCreator/itemCreator.js

```diff
@@ -193,6 +193,10 @@ define([
                     });
                 });
 
+                this.on('exit', function() {
+                    $(".item-editor-drop-area").remove();
```

Comment:
May be `$('.item-editor-item').empty()` could be more appropirate. @ssipasseuth  could you please validate ?

Reply:
`.item-editor-item` looks like a more suitable selector as it is higher level than `.item-editor-drop-area`.
Also, wouldn't it be better to scope the selector, by using the area broker for instance ?

Reply:
agree, @start-admin could you please test with `$('.item-editor-item', areaBroker.getItemPanel()).empty()` (see https://github.com/oat-sa/extension-tao-itemqti/blob/3f373fffcaaf9cc885d44e980a25641e5daf7141/views/js/controller/creator/index.js#L47)